sdk/spring/azure-active-directory-b2c-spring-boot-starter/README.md

```diff
@@ -0,0 +1,239 @@
+# Azure AAD B2C Spring Boot Starter client library for Java
+## Overview
+
+Azure Active Directory (Azure AD) B2C is an identity management service that enables you to customize and control how
+customers sign up, sign in, and manage their profiles when using your applications. Azure AD B2C enables these actions
+while protecting the identities of your customers at the same time.
+
+## Prerequisites
+
+The following prerequisites are required in order to complete the steps in this article:
+
+* A supported Java Development Kit (JDK). For more information about the JDKs available for use when developing on Azure, see <https://aka.ms/azure-jdks>.
+* [Apache Maven](http://maven.apache.org/), version 3.0 or later.
+* Azure subscription.
+
+If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
+
+## Getting started
+
+### Create the Active Directory instance
+
+1. Log into <https://portal.azure.com>.
+
+2. Click **+Create a resource**, then **Identity**, and then **Azure Active Directory B2C**.
+
+3. Enter your **Organization name** and your **Initial domain name**, record the **domain name** as your
+`${your-tenant-name}` and click **Create**.
+
+4. Select your account name on the top-right of the Azure portal toolbar, then click **Switch directory**.
+
+5. Select your new Azure Active Directory from the drop-down menu.
+
+6. Search `b2c` and click `Azure AD B2C` service.
+
+### Add an application registration for your Spring Boot app
+
+1. Select **Azure AD B2C** from the portal menu, click **Applications**, and then click **Add**.
+
+2. Specify your application **Name**, add `http://localhost:8080/home` for the **Reply URL**, record the
+**Application ID** as your `${your-client-id}` and then click **Save**.
+
+3. Select **Keys** from your application, click **Generate key** to generate `${your-client-secret}` and then **Save**.
+
+4. Select **User flows** on your left, and then **Click** **New user flow **.
+
+5. Choose **Sign up or in**, **Profile editing** and **Password reset** to create user flows
+respectively. Specify your user flow **Name** and **User attributes and claims**, click **Create**.
+
+## Examples
+### Configure and compile your app
+
+1. Extract the files from the project archive you created and downloaded earlier in this tutorial into a directory.
+
+2. Navigate to the parent folder for your project, and open the `pom.xml` Maven project file in a text editor.
+
+3. Add the dependencies for Spring OAuth2 security to the `pom.xml`:
+
+   ```xml
```

Comment:
Just about all of the individual library README.md files have dependency tag similar to the one below. Each one of them is missing the auto-update tags, which would go before and after the xml markdown tags. For example, the dependency below would have the following encapsulating the XML block.
`[//]: # ({x-version-update-start;com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;current})`
XML Tags and dependency tags 
`[//]: # ({x-version-update-end})`
Note: The format of these tags specifically prevents them from showing up when the user views the readme. [azure-core's readme.md is an example of this.](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/README.md). Notice that you don't see the tags unless you view the raw.

With the aforementioned being said, if this particular tag is supposed to be matching the library being built then its groupId should be com.azure and the version should be updated as well.

Reply:
Updated for all occurrences in the README files.